File: Daily Challenge/C/Random Number Game 7.2.2026.c
#include <stdio.h>

int main() {
    int n;
    scanf("%d", &n);

    int num[n];
    for (int i = 0; i < n; i++)
        scanf("%d", &num[i]);

    int x;
    scanf("%d", &x);
    int y = x;

    for (int i = 0; i < n; i++) {
        if (num[i] == x)
            x = x * 2;
    }

    if (x == y)
        printf("0");
    else
        printf("%d", x);

    return 0;
}

File: Daily Challenge/Java/Random Number Game 7.2.2026.java
import java.util.*;

public class Hello {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int[] num = new int[n];
        for (int i = 0; i < n; i++)
            num[i] = sc.nextInt();

        int x = sc.nextInt();
        int y = x;

        for (int v : num) {
            if (v == x)
                x *= 2;
        }

        System.out.print(x == y ? 0 : x);
    }
}